### PR TITLE
Docker files removed, Windows batch file added

### PR DESCRIPTION
**Pull request by sarupanda**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Hasith Kalpage
```
Comment (paralta) [inline LICENSE:3]: Do not forget to change this

File: hello_bot/helpers/html_helper.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+def html_bold(text):
```
Comment (paralta) [inline hello_bot/helpers/html_helper.py:1]: do we need this file?

File: hello_bot/helpers/spark_helper.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+__all__ = ['delete_webhook', 'delete_all_webhook',
```
Comment (paralta) [inline hello_bot/helpers/spark_helper.py:1]: Might be worth changing this file name to webex_helper.py ?

**Review by paralta — CHANGES_REQUESTED** [review]
LGTM, just some small changes :) 
Do we need to include `webexteamssdk` dir here?

File: hello_bot/helpers/html_helper.py (hunk 2 above)
Comment (sarupanda) [reply hello_bot/helpers/html_helper.py:1]: Don't know, but feel free to fix that in your code cleanup 😄

**Comment by sarupanda** [comment]
> LGTM, just some small changes :)
> Do we need to include `webexteamssdk` dir here?

So that would be the next commit: adding the git submodule for the webex teams sdk

**Pull request merged**